core/src/main/java/org/apache/carbondata/core/util/CarbonProperties.java

```diff
@@ -696,9 +697,6 @@ private void validateCarbonDataFileVersion() {
             CarbonCommonConstants.CARBON_DATA_FILE_DEFAULT_VERSION);
       }
     }
-    LOGGER.info("Carbon Current data file version: " + carbonProperties
```

Comment:
Log removal is intentional ? No need to log the current version?

Reply:
Added log